src/plugin/nerd_plugin.js

```diff
@@ -41,30 +42,30 @@ class NerdPluginVirtual extends EventEmitter {
 
     this.id = opts.id // not used but required for compatability with five
                       // bells connector.
-    this.auth = opts.auth
-    this.store = opts.store
+    this.auth = opts
+    this.store = opts._store
     this.timers = {}
 
-    this.info = opts.auth.info || {
-      precision: opts.auth.precision || 15,
-      scale: opts.auth.scale || 15,
-      currencyCode: opts.auth.currencyCode || '???',
-      currencySymbol: opts.auth.currencySymbol || '?'
+    this.info = opts.info || {
```

Comment:
I think it's cleaner to include in one object if you want to exclude one of the fields in LedgerInfo